src/Components/Nav.js

```diff
@@ -0,0 +1,346 @@
+import React, { useEffect, useState } from "react";
+import { mainData } from "../MockData/MainData.js";
+import styled from "styled-components";
+
+const Nav = ({ display, visibility }) => {
+  const [toggle, setToggle] = useState(false);
+  const [searchText, setSearchText] = useState("");
+  const [data, setData] = useState(mainData.searchData);
+  const excludeColumns = ["id", "imgUrl"];
+
+  const onToggle = () => {
+    setToggle(toggle === false ? true : false);
+  };
+
+  const handleChange = (value) => {
+    setSearchText(value);
+    filterData(value);
+  };
+
+  const filterData = (value) => {
+    const lowercasedValue = value.toLowerCase().trim();
+    if (lowercasedValue === "") {
+      setData(mainData.searchData);
+    } else {
+      const filteredData = mainData.searchData.filter((item) => {
+        return Object.keys(item).some((key) =>
+          excludeColumns.includes(key)
+            ? false
+            : item[key].toString().toLowerCase().includes(lowercasedValue)
+        );
+      });
+      setData(filteredData);
+    }
+  };
+
+  const useScroll = () => {
+    const [scroll, setScroll] = useState(0);
+
+    const onScroll = () => {
+      setScroll({
+        y: window.scrollY,
+      });
+    };
+
+    useEffect(() => {
+      window.addEventListener("scroll", onScroll);
+      return () => window.removeEventListener("scroll", onScroll);
+    }, []);
+
+    return scroll;
+  };
+
+  const { y } = useScroll();
+
+  return (
+    <NavContainer>
+      <div
+        className="toggleNav"
+        onFocus={onToggle}
+        onBlur={onToggle}
+        display={display}
+      >
+        <div></div>
+      </div>
+      <NavWrap>
+        <LogoWrap>
+          <img alt="logo" src="/images/logo.png" />
+        </LogoWrap>
+        <NavSearchWrap visibility={y > 350 ? 1 : 0}>
```

Comment:
```suggestion
        <NavSearchWrap visibility={y > 350}>
```
자체적으로 true false를 리턴하지 않을까요?